src/reducers/resourceHelpers.js

```diff
@@ -177,6 +177,9 @@ const removeFromKeyArray = (obj, key, value) => {
   obj[key] = [...obj[key]].filter((checkValue) => checkValue !== value)
 }
 
+// refactor this into more than one function;  move into separate file
+//   look at how we validate templates ... maybe use that as a model??
+//   (see actionCreators/templateValidationHelpers)
```

Comment:
I wanted to get tests passing and code written before refactoring.  I'm at that point now.